kombu/asynchronous/aws/connection.py

```diff
@@ -229,9 +229,11 @@ def get_status(self, operation, params, path='/', parent=None, verb='GET', callb
 
     def _on_list_ready(self, parent, markers, operation, response):
         service_model = self.sqs_connection.meta.service_model
+        protocol = service_model.protocol
```

Comment:
Could this be a class level static variable?

Reply:
No, since the `self.sqs_connection` itself is an instance variable rather than a static variable, We could make the `protocol` an instance variable as well, but there isn't much advantage, and I would rather not since we can easily access the protocol from the service model.